src/components/accordion/accordion.stories.mdx

```diff
@@ -164,6 +164,25 @@ Extra padding be be added top and bottom.
   </Story>
 </Preview>
 
+### Opening button
+A button can be passed in to replace the title and opening icon.
```

Comment:
As this scenario excludes using the validation, I think it would be good to add some guidance about avoiding using this configuration embedded in a form